readme.md

```diff
@@ -0,0 +1,154 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Bookstore React App</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [BOOKSTORE REACT APP] <a name="about-project"></a>
+
+**[Bookstore]** is a simple project which is made using React JS and it is adding, removing and storing the books into the local storage of the browser.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React js
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project is created using React JS .
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Adding Book]**
+- **[Removing Book]**
+- **[Storing the list of books in local storage]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
```

Comment:
- [ ] @Rifat8080 Great work on your documentation however I would suggest you add a link to your live demo r delete this entire section, this will make your readme be more professional by having all links that are working.